spec/vt-good-image-protocol.tex

```diff
@@ -111,6 +112,8 @@ \section{Terminal Emulator Requirements} % {{{
 Upper limits must be present for security reasons but can be varying by implementation.
 
 \todo{Have the ability to query resource limits?}
+\todo{What should happen when the number of images an application wants 
```

Comment:
Good point. Eviction in a deterministic priority should happen. I think I even mentioned that somewhere. I hope. It should be placed more prominent then.